packages/api-bzz-browser/src/index.ts

```diff
@@ -16,6 +18,10 @@ export class Bzz extends BaseBzz<Response> {
     super(window.fetch.bind(window), { ...cfg, url: new URL(url).href })
   }
 
+  protected normalizeStream(stream: ReadableStream): Readable {
```

Comment:
Rather than adding this method, could the stream type be defined on the class itself (https://github.com/MainframeHQ/erebos/pull/116/files#diff-ee8bd6642ded7aedea8cbd7b0b0978c8R136) please?

Reply:
Hmmm, I am not sure if I understand you here.

This is not meant as only type conversion for TS, but actual conversion to readable-stream. In Node nothing happens (only type conversion), but in browsers the WHATWG Readable Stream is converted to `readable-stream`. I don't think this is possible to do with type no?

Reply:
Yes sorry my bad I thought it was only setting the type, no need to change anything.